cluster/gce/gci/configure-helper.sh

```diff
@@ -1506,11 +1506,12 @@ function set_docker_options_non_ubuntu() {
 
    addockeropt "\"mtu\": 1460,"
    addockeropt "\"storage-driver\": \"overlay2\","
-
    echo "setting live restore"
    # Disable live-restore if the environment variable is set.
    if [[ "${DISABLE_DOCKER_LIVE_RESTORE:-false}" == "true" ]]; then
-      addockeropt "\"live-restore\": \"false\","
+      addockeropt "\"live-restore\": false,"
```

Comment:
are quotes removed intentionally?

Reply:
checked, likely not-quoted version works better

Reply:
yes, setting the string for boolean values is not working properly.